tests/unit/projection-test.js

```diff
@@ -1262,26 +1254,13 @@ module('unit/projection', function(hooks) {
       run(() => {
         store.push({
           data: {
-            id: BOOK_ID,
-            type: BOOK_CLASS_PATH,
-            attributes: {},
-            meta: {
-              projectionTypes: [BOOK_CLASS_PATH]
-            }
-          },
-          included: [
-            {
-              id: PUBLISHER_ID,
-              type: PUBLISHER_CLASS,
-              attributes: {
-                location: NEW_PUBLISHER_LOCATION,
-                owner: NEW_PUBLISHER_OWNER
-              },
-              meta: {
-                projectionTypes: [PUBLISHER_CLASS]
-              }
-            }
-          ]
```

Comment:
semantically this is probably the same change as just dropping the erroneous inclusion of `projectionTypes`; however, since the idea is to simulate an included record in a different payload I'd prefer to keep it structured that way.

Reply:
So, the idea is to always have book/projection of a book in the `data` and publisher in the `included`?

I was just trying to push the minimal required data to exercise the use case, in which case is an update to the publisher.